lib/uffizzi/cli/config.rb

```diff
@@ -17,40 +40,32 @@ def run(command, property, value)
         handle_set_command(property, value)
       when 'delete'
         handle_delete_command(property)
-      else
-        puts "#{command} is not a uffizzi config command"
       end
     end
 
-    private
-
     def handle_list_command
       ConfigFile.list
     end
 
     def handle_get_command(property)
-      if property.nil?
-        puts 'No property provided'
-        return
-      end
+      return Uffizzi.ui.say('No property provided') if property.nil?
+
       option = ConfigFile.read_option(property.to_sym)
-      puts option unless option.nil?
+      Uffizzi.ui.say(option) unless option.nil?
```

Comment:
So if the option does not exist it will just do nothing?

Reply:
there is line
`Uffizzi.ui.say("The option #{option} doesn't exist in config file") if data[option].nil?`
in read_option method

Reply:
Sounds like a SRP violation